ngx-fudis/projects/ngx-fudis/src/lib/components/notification/notification.component.html

```diff
@@ -1,15 +1,17 @@
-<div class="fudis-notification fudis-notification__{{ variant }}">
+<article
+  class="fudis-notification fudis-notification__{{ variant }}"
+  [attr.aria-label]="_translateService.getTranslations()().ICON.ATTENTION"
```

Comment:
Maybe this Attention text should not be under `ICON` anymore in our translation keys